docs/user_guide/features/csg_and_tessellated_module.rst

```diff
@@ -308,7 +308,121 @@ Tesselated geometries can also be combined with the primitives
    Tesselated Geometry sampling using Modulus Sym: Stanford bunny
 
 
+Signed Distance Fields (SDF) of Geometry objects
+-------------------------------------------------
 
+Mathematically, signed distance field or signed distance function (SDF) is defined as
+the orthonogal distance of a given point to the boundary / surface of a geometric shape.
+It is widely used to describe the geometry in mathematics, rendering, and similar
+applications. In physics-informed learning, it is also used to represent
+`geometric inputs to the neural networks <https://www.research.autodesk.com/app/uploads/2023/03/convolutional-neural-networks-for.pdf_rectr0tDKzFYVAAJe.pdf>`_.
 
+Modulus' geometry module (CSG and Tesselation) compute the SDF (and its derivatives) on
+the points sampled in the interior which can be used in the training pipelines.
+Additionally, the SDF can be computed on arbitrary points using the ``.sdf`` attribute.
+ 
+Modulus also provides a utility to recover the STL geometry from the SDF. This utility
```

Comment:
Repetitive lines, just change it to "Modulus also provides a utility to recover the STL geometry from the SDF using Marching Cubes".